templates/simple/amq/simple-fuse-activemq-spring/src/main/java/com/mycompany/templates/camel/activemq/OrderGenerator.java

```diff
@@ -1,45 +0,0 @@
-/**
```

Comment:
Why did you removed this file? it is due to that the test is failing. The project is not compiling due to this error